Makefile.am

```diff
@@ -245,10 +245,6 @@ jslint-ui: $(top_builddir)/install/ui/src/libs/loader.js
 	rm js/libs;						\
 	rmdir js
 
-jslint-ui-test:
```

Comment:
jslint-ui-tests shouldn't be removed. Some of the tests remains there so it should stay and also should be kept in jslint target.